src/unistd.rs

```diff
@@ -373,13 +373,37 @@ pub fn sleep(seconds: libc::c_uint) -> c_uint {
     unsafe { libc::sleep(seconds) }
 }
 
+pub fn execvpe(filename: &CString, args: &[CString], env: &[CString]) -> Result<()> {
+    use std::env;
+    use std::ffi::OsString;
+    use std::ffi::OsStr;
+    use std::os::unix::ffi::OsStrExt;
+
+    if filename.as_bytes().iter().find(|c| **c == b'/').is_some() {
+        return execve(filename, args, env);
+    }
+
+    let paths = match env::var_os("PATH") {
+        Some(val) => val,
+        None => OsString::from("/usr/local/bin:/bin:/usr/bin"),
```

Comment:
I took the same value as musl libc.
If you want to stick to the POSIX standard, it is only "/bin/:usr/bin" according to confstr(3).